docs/ARMInstallation.md

```diff
@@ -2,11 +2,11 @@
 This article details provisioning and installation of the IoMT FHIR Connector for Azure using an ARM template.
 
 ## ARM Template Provisioning
-<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FMicrosoft%2Fiomt-fhir%2Fmaster%2Fdeploy%2Ftemplates%2Fdefault-azuredeploy.json" target="_blank">
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FMicrosoft%2Fiomt-fhir%2Fmaster%2Fdeploy%2Ftemplates%2Fflexible-azuredeploy.json" target="_blank">

```

Comment:
Let me know if we don't want to keep this anymore. At first, I was planning on removing it, but from the readme it seemed like we want to keep this as a flexible template.

Reply:
I think it would be better to keep this page but update it to reference the default-managed-identity-azure template.  Basically we should use the MI template as the default and the customer can choose if they want to do the ARM deployment or the Sandox for it.

Reply:
As follow up, we would functionality delete this and rename the ARMInstallationManagedIdentity to this file name update links as needed.

Reply:
Yeah that was what I gathered earlier as well, however in the ReadMe I read this and figured we'd want to have this as an option too if using another identity provider for whatever reason:
Open Source Deployment - Flexible: Describes how to deploy the IoMT FHIR Connector for Azure us
ing another FHIR server or identity provider.
If not, then yes makes sense to only just have the MI one(renamed to this) as have that be the only reference.